cross/salt/Makefile

```
@@ -0,0 +1,25 @@
+PKG_NAME = salt
+PKG_VERS = 3003.3
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/s/$(PKG_NAME)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = 
+
+HOMEPAGE = https://pypi.python.org/pypi/salt
+COMMENT  = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with tho$
```

Did your editor cutoff the end?

```suggestion
COMMENT  = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
```

---

fixed.